tests/test_script_helper.py

```diff
@@ -225,318 +287,312 @@ def generate_random_synapses_locations(cell,syn_filename,num_syns=10000,seed=2):
-    MODEL_FILE_REDUCED = reduced_model_file
-    SYNAPSE_FILE = synapse_file
-    MORPHOLOGY_FILE = orig_morphology_file
-    MANUAL_TOTAL_NSEGS = manual_total_nsegs
-    VOLTAGEFILE = voltage_file
+    h.celsius = celsius  # needs to be set for reduction to work properly
+
+    reduction_frequency = int(reduction_frequency)
     print('---------------sim data --------------------------')
-    print("REDUCTION_FREQUENCY " + str(REDUCTION_FREQUENCY))
-    print("MODEL_FILE          " + str(MODEL_FILE))
-    print("MODEL_FILE_REDUCED  " + str(MODEL_FILE_REDUCED))
-    print("SYNAPSE_FILE        " + str(SYNAPSE_FILE))
-    print("MORPHOLOGY_FILE     "  + str(MORPHOLOGY_FILE))
-    print("MANUAL_TOTAL_NSEGS  " + str(MANUAL_TOTAL_NSEGS))
-    print("VOLTAGEFILE         "  + str(VOLTAGEFILE))
+    print("REDUCTION_FREQUENCY %s" % reduction_frequency)
+    print("MODEL_FILE          %s" % model_file)
+    print("MODEL_FILE_REDUCED  %s" % model_file_reduced)
+    print("SYNAPSE_FILE        %s" % synapse_file)
+    print("MORPHOLOGY_FILE     %s" % morphology_file)
+    print("MANUAL_TOTAL_NSEGS  %s" % manual_total_nsegs)
+    print("VOLTAGEFILE         %s" % voltage_file)
     print('vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv\n\n')
 
-    if create_type in ['almog','hay','bbpnew', 'bbpactive','allen','human']:
-        cwd = os.getcwd()
-        os.chdir(MODEL_FILE[:MODEL_FILE.rindex('/')])
-        process = subprocess.Popen(['nrnivmodl','mod'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        print('compiling mod files')
-        stdout, stderr = process.communicate()
-        #print(stdout)
-        #print(stderr)
-        h.nrn_load_dll("x86_64/.libs/libnrnmech.so.0")
-        h.nrn_load_dll("x86_64/.libs/libnrnmech.0.so")
-        os.chdir(cwd)
+    if create_type in ('almog', 'hay', 'bbpnew', 'bbpactive', 'allen', 'human'):
+        with cd(model_file[:model_file.rindex('/')]):
+            process = subprocess.Popen(['nrnivmodl', 'mod'],
+                                       stdout=subprocess.PIPE,
+                                       stderr=subprocess.PIPE)
+            print('compiling mod files')
+            stdout, stderr = process.communicate()
+            #print(stdout)
+            #print(stderr)
+            h.nrn_load_dll("x86_64/.libs/libnrnmech.so.0")
+            h.nrn_load_dll("x86_64/.libs/libnrnmech.0.so")
 
     if create_type == 'hay':
-        loadtemplate(MODEL_FILE[:MODEL_FILE.rindex('/')]+'/L5PCbiophys3.hoc')
-    if create_type =='allen':
-        loadtemplate(MODEL_FILE[:MODEL_FILE.rindex('/')]+'/AllenBiophys.hoc')
+        loadtemplate(model_file[:model_file.rindex('/')] + '/L5PCbiophys3.hoc')
+    elif create_type == 'allen':
+        loadtemplate(model_file[:model_file.rindex('/')] + '/AllenBiophys.hoc')
+
     h.load_file("import3d.hoc")
-    loadtemplate(MODEL_FILE)
-    
+    loadtemplate(model_file)
 
     # creates original cell with synapses, and reduces it with NeuroReduce
```

Comment:
can you please change NeuroReduce to Neuron_Reduce

Reply:
sure